extra_data/components.py

```diff
@@ -1394,6 +1373,56 @@ def _make_image_index(self, tids, inner_ids, inner_name='pulse'):
         )
 
 
+@multimod_detectors
+class LPD1M(LPDBase, XtdfDetectorBase):
+    """An interface to LPD-1M data.
+
+    Parameters
+    ----------
+    data: DataCollection
+      A data collection, e.g. from :func:`.RunDirectory`.
+    modules: set of ints, optional
+      Detector module numbers to use. By default, all available modules
+      are used.
+    detector_name: str, optional
+      Name of a detector, e.g. 'FXE_DET_LPD1M-1'. This is only needed
+      if the dataset includes more than one LPD detector.
+    min_modules: int
+      Include trains where at least n modules have data. Default is 1.
+    parallel_gain: bool
+      Set to True to read this data as parallel gain data, where high, medium
+      and low gain data are stored sequentially within each train. This will
+      repeat the pulse & cell IDs from the first 1/3 of each train, and add gain
+      stage labels from 0 (high-gain) to 2 (low-gain).
+    """
+    _source_re = re.compile(r'(?P<detname>.+_LPD1M.*)/DET/(?P<modno>\d+)CH')
+    module_shape = (256, 256)
+
+
+@multimod_detectors
+class LPDMINI(LPDBase, XtdfDetectorBase):
```

Comment:
Is it common to write LPD MINI in all capitals? If not, let's call the class `LPDMini`.

JUNGFRAU is in block capitals because [that's how it's written](https://www.psi.ch/en/lxn/jungfrau) - it's an incredibly tortured acronym.

Reply:
I see, I went with all capitals as other detectors we all capitals.  
I'm happy to make it camelCased.